src/core.c/Supply-factories.pm6

```diff
@@ -86,11 +86,16 @@
 
         my class IntervalCancellation is Cancellation {
             has $!delegate;
+#?if moar
             has atomicint $!cancelled;
+#?endif
+#?if !moar
+            has int $!cancelled;
+#?endif
             has $!lock;
 
             submethod BUILD(--> Nil) {
-                $!cancelled ⚛= 0;
+                $!cancelled = 0;
```

Comment:
`$!cancelled`'s value should already be `0`. That's the default for `atomicint` as a native int.